src/rpc/rpc.js

```diff
@@ -732,8 +733,7 @@ class RPC extends EventEmitter {
     /**
      *
      */
-    _on_message(conn, msg_buffer) {
-        const msg = RpcRequest.decode_message(msg_buffer);
+    _on_message(conn, msg) {
```

Comment:
I do need to learn this JSDoc thingy (mainly syntax)

Reply:
@guy  
I cannot use RpcBaseConnection here because it is not imported into the file.
If I import it then it is an unused variable.

Any suggestions?

Reply:
oops, I accidently wrote `@type` instead of `@param`. thanks for fixing, resolving.

Reply:
About RpcBaseConn you can use it like this example - 
https://github.com/noobaa/noobaa-core/blob/c3cce9ba10174b0ab41481fe1aed204b8fd02157/src/sdk/map_api_types.js#L5